solarforecastarbiter/validation/tasks.py

```diff
@@ -280,3 +371,70 @@ def immediate_observation_validation(access_token, observation_id, start, end,
     observation_values.update(quality_flags)
     session.post_observation_values(observation_id, observation_values,
                                     params='donotvalidate')
+
+
+DAILY_VALIDATION_FUNCS = {
+    'ghi': validate_daily_ghi,
+    'dc_power': validate_daily_dc_power,
+    'ac_power': validate_daily_ac_power
+}
+
+
+def _daily_validation(session, observation, start, end, base_url):
+    observation_values = session.get_observation_values(
+        observation.observation_id, start, end)
+    value_series = observation_values['value']
+    if len(value_series.dropna()) < 10:
```

Comment:
How did you decide on 10? 

Hourly data in the winter might have less than 10 data points if only uploaded during the day. Maybe it shouldn't work in that case, just pointing it out.

Reply:
From https://github.com/SolarArbiter/solarforecastarbiter-core/issues/71#issuecomment-486274741

>Checks requiring a time series: detect_stale_values, detect_interpolation, detect_clipping. For these, I suggest requiring at least 24 hours(?) of data? Or 10+ data points. With shorter time series the code should run but the output is probably not useful.

The other checks should already have been done by the immediate validation, so OK to fail when < 10

Reply:
Since the functions are windows based on count, I figured I would go with 10 vs 24 hrs